lib/mix/lib/mix/release.ex

```diff
@@ -91,14 +91,15 @@ defmodule Mix.Release do
     {erts_source, erts_lib_dir, erts_version} = erts_data(include_erts)
 
     deps_apps = Mix.Project.deps_apps()
-    loaded_apps = apps |> Keyword.keys() |> load_apps(deps_apps, %{}, erts_lib_dir, [], :root)
+    loaded_apps = load_apps(apps, deps_apps, %{}, erts_lib_dir, [], apps)
```

Comment:
I know this might not be a convention in the Elixir codebase, but I think it might be nice to be more explicit on what this is?

```suggestion
    loaded_apps = load_apps(apps, deps_apps, %{}, erts_lib_dir, [], _overrides = apps)
```